twittermap/public/javascripts/common/services.js

```diff
@@ -2,6 +2,7 @@ angular.module('cloudberry.common', [])
   .factory('cloudberryConfig', function(){
     return {
       ws: config.wsURL,
+      sentimentAnalysisEnabled: true,
```

Comment:
you can track the code inside the `views/twittermap/index.scala.html` and the `controllers/TwitterMapApplication.scala` to see how the `ws` address is set.